tests/4.5/taskloop/test_taskloop_collapse.c

```diff
@@ -0,0 +1,58 @@
+//------------------------------------------- test_taskloop_collapse.c --------------------------------------//
+//
+// OpenMP API Version 4.5 September 2015
+//
+// This test checks the 'taskloop' directive with the 'collapse' clause specified.
+// The 'taskloop' construct parallelize loops with independent iterations by creating tasks. 
+// It allows for efficient parallel execution of loop iterations by distributing them among multiple threads. 
+// The 'collapse' clause ensures that multiple nested loops into a single loop.      
+//----------------------------------------------------------------------------------------------------------//
+
+#include <stdio.h>
+#include <omp.h>
+#include "ompvv.h"
+
+#define NUM_THREADS 100
+#define NUM_COLLAPSE 2
+
+int test_taskloop_collapse() {
+
```

Comment:
For any other value of WEEK the test will fail. 

In other thoughts, there are better ways to test collapse without requiring the use of atomic.

Reply:
https://github.com/SOLLVE/sollve_vv/pull/782/commits/24882279d1b6bd4bd0a2f3639ee75976d06dbf93